lua/entities/gmod_wire_expression2/core/custom/cl_cfc_e2_lib.lua

```diff
@@ -1,46 +1,103 @@
 -- PvP Functions
```

Comment:
This file will need to be split up as the other files are pulled out into their own PR.